WordPress/Classes/ViewRelated/Post/PostEditorAnalyticsSession.swift

```diff
@@ -55,6 +60,7 @@ struct PostEditorAnalyticsSession {
         let outcome = self.outcome ?? endOutcome
         let properties = [
             Property.outcome: outcome.rawValue,
+            Property.template: pageTemplateApplied,
```

Comment:
@koke for analytics if `pageTemplateApplied` is blank should we exclude it here or is that handled in tracks?

Reply:
I believe it's better to exclude it than to send an empty string. IIRC that would translate to a NULL value on the backend when querying.